source/guide/3.2/plugin-hostpool.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: bt_wiki
+title: Host Pool Plugin
+category: Plugins
+publish: true
+abstract: Cloudify Host Pool plugin description and configuration
+pageord: 600
+---
+{%summary%}
+{%endsummary%}
+
+
+# Description
+
+The Host Pool plugin is an interface to [Host Pool service](https://github.com/cloudify-cosmo/cloudify-host-pool-service).
```

Comment:
i would not mix the word plugin and interface as if they have the same meaning.
instead maybe just say that this plugin interacts with the host pool service.